### DISCUSSION TITLE
Member only discussion area

### DISCUSSION THREAD

**Discussion by glennawatson** [dotnet-foundation/Home#48] [General Discussions]
Some members need a space to air their opinions about topics without the heated discussion from the outside world.

Given more intense debate lately, setting up some member-only discussion boards in GitHub to discuss feedback would be helpful.

Additionally, some members are asking about the value of membership. If there is more community space where members can discuss their project issues, it could add value without fear it being all public.

Given the recent shutdown of the other GitHub discussions, which were member-only, something might be worth bringing back.

**Comment by glennawatson** [comment]
Full credit to @Perksey for this issue

**Comment by ADefWebserver** [comment]
As a member, I would like us, as an organization, to do Better.
To be Humane and Kind, and Understanding of each other.

Yes, we can still disagree and debate, but #39 is strait-up 'Toxic' 

Now, if that is only 'outsiders' then ok I can accept that and still be happy to be a member of this organization.
I am making this post because I want to see what the response is...

**Reply by JimBobSquarePants** [reply]
Dude. Valid criticism of process and resting accountability is far from “Toxic” and viewing everything as such indicates both a lack of empathy on your part and a juvenile interpretation of events. There’s been no bullying, only a valid expression of anger and disappointment.

**Reply by ADefWebserver** [reply]
> Dude. Valid criticism of process and resting accountability is far from “Toxic” and viewing everything as such _**indicates both a lack of empathy**_ on your part and a **juvenile interpretation** of events. There’s been no bullying, only a valid expression of anger and disappointment.

I am going to politely disagree with your entire statement. 

- I feel your words that I have put in **Bold** are symptomatic of my point of being Bullying.
- Your words I have put in **_ Bold italics_** are [gaslighting ](https://www.verywellmind.com/is-someone-gaslighting-you-4147470)(that is also another form of Bullying)  

It is statements like this, toward each other, that I consider 'Toxic' and that I would like us to please-stop-doing.

Thank You

**Reply by glennawatson** [reply]
@ADefWebserver straight up question, what do you feel is toxic about #39?- I will be honest I think the earlier discussions were on point and justified, there were some questions definitely not answered in the apology (and hey she made a mistake with the PR all done and dusted), but I think there was some necessary points raised. It felt a little bit of the same points going back between two parties later on though.

**Reply by ADefWebserver** [reply]
> @ADefWebserver straight up question, what do you feel is toxic about #39?-

Several things...

- Asking the person to resign (no basis for this - it was a 4 line PR)
- Statements calling it a non-apology (this is how you can justify continued criticism)
- Post after post re-criticizing the person for no valid reason. Yes it was wrapped in 'we are displeased' but when you keep going until the person resigns it is by definition Bullying. 

Just because someone is right does not mean that their "behavior' is not Bullying.

**Reply by glennawatson** [reply]
Fair enough,

I think Dan was asking more for the resignation due to the GitHub ownership changes, and general issues with ownership that have been ongoing rather than the PR, but he can comment on that more if he wishes. If it was just about the PR it is definitely an overreaction.

The non-apology yeah I agree with you, she apologised for the PR, also she sent me a private apology which I later accepted as well so that should have been dusted by that time. I let people know all this on my Twitter feed and stuff.

The discussion needed to be had though in regards to the other problems which I guess they are now.

**Reply by JimBobSquarePants** [reply]
Nobody asked anyone to resign over a PR and to accuse someone of doing so is dangerous misrepresentation.

**Reply by dansiegel** [reply]
The PR didn't make my top 10. When you consider how many maintainers just found out their organizations had been victim to the dnfadmin account being misused to forcefully take over the org without anyone knowing. When you look at all of the other issues which were either not addressed or there was a vailed attempt to justify. There was a blatant violation of trust, something that many maintainers including myself would not easily get past without a change in leadership. There is also the matter of issues brought up by @rlittlesii which are serious in their own right. Ultimately those reasons were why I asked for her to resign.

The PR was the straw that broke the camels back, but this was about a lot more serious issues than that single PR. If this was merely about the PR then all @clairernovotny ever needed to do was respond that she recognized that she had overstepped and it wouldn't happen again. Keep in mind you do not know me, I have known @clairernovotny for a long time and always considered her a friend. It was very tough for me to have to come out and publicly criticize someone who I have so much respect for, and I did not make such a public request lightly.

**Comment by MeikTranel** [comment]
I'm still a little dumb founded why exactly we have to discuss/debate introducing something we already had until two weeks ago, when somehow admins just disabled https://github.com/orgs/dotnet-foundation/teams/members ? 

I dont need a new msteams infrastructure or something. We already had an adequate solution. Just re-enable it and this entire discussion can be resolved.
